messages/scan.json

```diff
@@ -3,6 +3,7 @@
   "rulesetFlagDescription": "ruleset to use for scanning",
   "filepathFlagDescription": "file path to examine",
   "reportFlagDescription": "file to save output report",
+  "customJarDescription": "location of jar file with custom rules. Please make sure jar includes compiled class of the custom rule and an XML file with the rule definition. You can refer to PMD's Custom Rule definition for more information. https://pmd.github.io/latest/pmd_userdocs_extending_writing_pmd_rules.html",
```

Comment:
Can we not just remove this command entirely, now that we've got `scanner:run`?

Reply:
I've been using this command for sanity testing. We can remove it once add command is ready.